engine/src/main/java/io/camunda/zeebe/engine/state/ZbColumnFamilies.java

```diff
@@ -45,6 +45,7 @@
   JOBS,
   JOB_STATES,
   JOB_DEADLINES,
+  JOB_BACKOFF,
```

Comment:
:x: We need to add the new enum at **the end** of the class.

This is an import implementation detail :exclamation: 
However, we should remove this trap eventually :see_no_evil: I'm sorry.

Reply:
Done